README.md

```diff
@@ -7,25 +7,25 @@ Windows Presentation Foundation (WPF) is a UI framework for building Windows des
 
 WPF applications are based on a vector graphics architecture. This enables applications to look great on high DPI monitors, as they can be infinitely scaled. WPF also includes a flexible hosting model, which makes it straightforward to host a video in a button, for example. The visual designer provided in Visual Studio makes it easy to build WPF application, with drag-in-drop and/or direct editing of XAML markup.
 
+As of .NET 6.0 WPF supports ARM64. 
+
 See the [WPF Roadmap](roadmap.md) to learn about project priorities, status and ship dates.
 
 [WinForms](https://github.com/dotnet/winforms) is another UI framework for building Windows desktop applications that is supported on .NET (6.0.x/5.0.x/3.1.x). WPF and WinForms applications only run on Windows. They are part of the `Microsoft.NET.Sdk.WindowsDesktop` SDK. You are recommended to use the most recent version of [Visual Studio](https://visualstudio.microsoft.com/downloads/) to develop WPF and WinForms applications for .NET.  
 
-WPF for ARM64 is new for .NET 6.0 and is supported by NET 6.0 Preview 1 and later. 
-
-To build the WPF repo and contribute features and fixes for .NET 6.0, the most recent [Visual Studio Preview](https://visualstudio.microsoft.com/vs/preview/) is required.  
+To build the WPF repo and contribute features and fixes for .NET 6.0, please read our [contributing guide](Documentation/contributing.md).  
```

Comment:
@kant2002  we will still need VS2022 Preview to be able build on .NET 7. 
May be this shall be updated to 
```suggestion
To build the WPF repo and contribute features and fixes for .NET 7.0,  [Visual Studio 2022 Preview](https://visualstudio.microsoft.com/vs/preview/) is required.
```